File: app/src/main/java/com/example/focusstartsecondpart/features/profiles/presentation/ProfileActivityPresenter.java
package com.example.focusstartsecondpart.features.profiles.presentation;

import com.example.focusstartsecondpart.features.BasePresenter;

public class ProfileActivityPresenter extends BasePresenter<ProfileView> {

    public ProfileActivityPresenter(){

    }

    @Override
    public void onViewReady(){
        view.showProgress();
        view.showProfile();
        view.hideProgress();
    }
}

File: app/src/main/java/com/example/focusstartsecondpart/features/profiles/presentation/ProfileActivityPresenter.kt
package com.example.focusstartsecondpart.features.profiles.presentation

import com.example.focusstartsecondpart.App.BasePresenter

class ProfileActivityPresenter(private val profile: ArrayList<String>) : BasePresenter<ProfileView>() {

    public override fun onViewReady() {
        view.showProgress()
        view.showProfile(profile[0], profile[1], profile[2])
        view.hideProgress()
    }
}
